contents/teams/feature-flags/objectives.mdx

```diff
@@ -1,58 +1,53 @@
-- [Behavioral cohort targeting for flags](https://github.com/PostHog/posthog/issues/14796) (RFC [here](https://github.com/PostHog/product-internal/pull/683/)) (@dmarticus)
-- [Feature management](https://github.com/PostHog/posthog/issues?q=is:open+is:issue+label:feature/feature-management) (@havenbarnes)
-- [Automatically manage flag staleness](https://github.com/PostHog/posthog/issues/16497) (@haacked)
-- Customer research-driven features (TBD based on the findings of the market research and customer interviews)
-
-#### 3. Strategic research
-_Owners: @havenbarnes, @annikaschmid_
-- Identify and analyze key user segments:
-  1. Current active users
-  2. Competitor users
-  3. Custom solution implementers
-  4. Potential users unaware of feature flagging
-- Conduct customer interviews to identify gaps and growth opportunities
-- Document findings and recommendations for product roadmap.
-- More context [here](https://github.com/PostHog/posthog/issues/14257)
-
-#### 4. SDK Spike
-_Owners: @dmarticus, @havenbarnes, @haacked_
-- [Evaluate feasibility and tradeoffs around adding feature flags to the Java SDK](https://github.com/PostHog/posthog/issues/16419)
-- Investigate improving the Rust SDK (add feature flag evaluation, [implement the suggestions from the open issues](https://github.com/PostHog/posthog-rs/issues?q=sort:updated-desc+is:issue+is:open) etc.)
-- [Research .NET SDK demand + feasibility](https://github.com/PostHog/posthog/issues/16339)
-- Fix open issues in the Python and Node SDKs: ([Python](https://github.com/PostHog/posthog-python/issues?q=sort:updated-desc+is:issue+is:open), [Node](https://github.com/PostHog/posthog-js-lite/issues?q=sort:updated-desc+is:issue+is:open+label:nodejs))
-
-#### 5. User Experience Improvements
-_Owners: @dmarticus, @havenbarnes, @haacked_
-Low-hanging, medium-impact, and largely context-free fruit that can be picked up by any team member during available capacity:
-
-- UI Changes
-  - [Consolidate boolean and multivariate flag interfaces](https://github.com/PostHog/posthog/issues/26995)
-  - [Show flag evaluation results more clearly in the flag editor UI](https://github.com/PostHog/posthog/issues/26995)
-  - [Improve the experience of syncing flags between environments](https://github.com/PostHog/posthog/issues/21788)
-  - Anything else from the [current feature flag backlog](https://github.com/orgs/PostHog/projects/112/views/2)
+## Q2 2025 Objectives
+
+### Flag Stability & Consistency, and Resiliency
+*Goal: Create a highly stable, consistent, and resilient feature flag system capable of supporting 10x growth*
+
+- Scale `/flags` service
+   - Complete 100% worldwide rollout of `/flags` endpoint to all customers
+   - Measure and validate improved query performance metrics
+   - Document infrastructure cost savings achieved
+   - Stretch: Investigate feasibility of decoupling flag evaluation from Postgres entirely
+
+- Standardize SDK Error Handling and UX
```

Comment:
I'd like to add the goal you and I discussed with moving more client functionality into the server. Specifically, the experimentation stuff (aka proposal 2 in https://github.com/PostHog/posthog/issues/30466). To me, it's a top-level goal of "correctness", but it also relates to system stability and standardizing SDKs.

Reply:
added!